packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/StakeExchangeForm/StakeExchangeForm.tsx

```diff
@@ -116,6 +119,12 @@ const StakeExchangeForm = ({
         ethChainId,
       });
 
+      await createStakeTransaction.mutate({
+        id: '1',
+        transaction_hashes: [txReceipt.transactionHash],
```

Comment:
Why make this an array? If there is only 1 txnHash we can simplify the backend queries (search for equality rather than `IN` array)

Reply:
To backfill values we won't need (and we shouldn't) use the API since we can fetch txns directly from the chain (no need to verify). I think we should write a script which will be basically like a migration but more similar to the view count recovery you did recently.

Reply:
How about I convert it to only take in a single transaction, but we still go with the manual input method?

This way the code will be cleaner and it will still take less time than writing the script?

Reply:
The script would be pretty much exactly the same as your `createStake` function but with some additional block fetching/parsing which is quite simple. Given that there are only 100 txns those all fit into a single txn so there is no complicated deployment plan or difficult failure scenarios.

That said, if you think it will be faster for you to manually call the API that's up to you.

Reply:
Ok, converted the createStakeTransaction to only take in a single transaction at a time. Maybe sticking to the manual input is the best for the small number of transactions.